ansible/roles/test/files/ptftests/arista.py

```diff
@@ -375,6 +375,27 @@ def verify_bgp_neigh_state(self, dut=None, state="Active"):
                     self.fails.add('Verify BGP %s neighbor: Object missing in output' % ver)
         return self.fails, bgp_state
 
+    def change_neigh_lag_state(self, lag, is_up=True):
+        state = ['shut', 'no shut']
+        self.do_cmd('configure')
+        self.do_cmd('interface %s' % lag)
+        self.do_cmd('%s' % state[is_up])
```

Comment:
you don't need the template here.

Reply:
Didn't get you. Are you referring to passing the state string directly? If yes, this was a feedback I received from Qi in my earlier PR to pass in a  bool to avoid misspelling name.

Reply:
You can write
`self.do_cmd(state[is_up])`
instead of
`self.do_cmd('%s' % state[is_up])`

Reply:
done